home-open-api/src/main/resources/yaml/models/contact.yaml

```diff
@@ -63,12 +63,12 @@ CreateContact:
       default: false
       type: boolean
       example: false
-  example:
-    {
-      "type": "email",
-      "main": false,
-      "email": "test.put.messages@gmail.com"
-    }
+#  example:
```

Comment:
Why we left it commented?

Reply:
These parameters do not work correctly and do not allow you to use "create" and "update" in contact. We have not found a solution yet.